CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set(MMDEPLOY_TARGET_DEVICES
         "cpu" CACHE STRING "MMDeploy's target devices")
 set(MMDEPLOY_TARGET_BACKENDS "" CACHE STRING "MMDeploy's target inference engines")
 set(MMDEPLOY_CODEBASES "all"  CACHE STRING "select OpenMMLab's codebases")
+set(MMDEPLOY_BUILD_CSHARP_EXTERN "build MMDeployExtern.dll for c# api" OFF)
```

Comment:
To keep the naming consistent, use `MMDEPLOY_BUILD_SDK_CSHARP_API`, and it's "C# API"